proposals/simd/webassembly-opcodes.md

```diff
@@ -0,0 +1,275 @@
+| `i16x8.replaceLane(a: v128, i: LaneIdx8, x: i32) -> v128` | [i16x8.replaceLane](portable-simd.md#replace-lane-value) |
+| `i16x8.add(a: v128, b: v128) -> v128` | [i16x8.add](portable-simd.md#integer-addition) |
+| `i16x8.sub(a: v128, b: v128) -> v128` | [i16x8.sub](portable-simd.md#integer-subtraction) |
+| `i16x8.mul(a: v128, b: v128) -> v128` | [i16x8.mul](portable-simd.md#integer-multiplication) |
+| `i16x8.neg(a: v128) -> v128` | [i16x8.neg](portable-simd.md#integer-negation) |
+| `i16x8.addSaturate_s(a: v128, b: v128) -> v128` | [s16x8.addSaturate](portable-simd.md#saturating-integer-addition) |
+| `i16x8.addSaturate_u(a: v128, b: v128) -> v128` | [u16x8.addSaturate](portable-simd.md#saturating-integer-addition) |
+| `i16x8.subSaturate_s(a: v128, b: v128) -> v128` | [s16x8.subSaturate](portable-simd.md#saturating-integer-subtraction) |
+| `i16x8.subSaturate_u(a: v128, b: v128) -> v128` | [u16x8.subSaturate](portable-simd.md#saturating-integer-subtraction) |
+| `i16x8.shl(a: v128, y: i32) -> v128` | [i16x8.shiftLeftByScalar](portable-simd.md#left-shift-by-scalar) |
+| `i16x8.shr_s(a: v128, y: i32) -> v128` | [s16x8.shiftRightByScalar](portable-simd.md#right-shift-by-scalar) |
+| `i16x8.shr_u(a: v128, y: i32) -> v128` | [u16x8.shiftRightByScalar](portable-simd.md#right-shift-by-scalar) |
+| `i16x8.eq(a: v128, b: v128) -> b16x8` | [i16x8.equal](portable-simd.md#equality) |
+| `i16x8.ne(a: v128, b: v128) -> b16x8` | [i16x8.notEqual](portable-simd.md#non-equality) |
+| `i16x8.lt_s(a: v128, b: v128) -> b16x8` | [s16x8.lessThan](portable-simd.md#less-than) |
+| `i16x8.lt_u(a: v128, b: v128) -> b16x8` | [u16x8.lessThan](portable-simd.md#less-than) |
+| `i16x8.le_s(a: v128, b: v128) -> b16x8` | [s16x8.lessThanOrEqual](portable-simd.md#less-than-or-equal) |
+| `i16x8.le_u(a: v128, b: v128) -> b16x8` | [u16x8.lessThanOrEqual](portable-simd.md#less-than-or-equal) |
+| `i16x8.gt_s(a: v128, b: v128) -> b16x8` | [s16x8.greaterThan](portable-simd.md#greater-than) |
+| `i16x8.gt_u(a: v128, b: v128) -> b16x8` | [u16x8.greaterThan](portable-simd.md#greater-than) |
+| `i16x8.ge_s(a: v128, b: v128) -> b16x8` | [s16x8.greaterThanOrEqual](portable-simd.md#greater-than-or-equal) |
+| `i16x8.ge_u(a: v128, b: v128) -> b16x8` | [u16x8.greaterThanOrEqual](portable-simd.md#greater-than-or-equal) |
+
+## `v32x4` operations
+| WebAssembly | Portable SIMD |
+|:------------|:--------------|
+| `v32x4.select(s: b32x4, t: v128, f: v128) -> v128` | [v32x4.select](portable-simd.md#lane-wise-select) |
+| `v32x4.swizzle(a: v128, s: LaneIdx4[4]) -> v128` | [v32x4.swizzle](portable-simd.md#swizzle-lanes) |
+| `v32x4.shuffle(a: v128, b: v128, s: LaneIdx8[4]) -> v128` | [v32x4.shuffle](portable-simd.md#shuffle-lanes) |
+| `v32x4.load1(addr, offset) -> v128` | [v32x4.load1](portable-simd.md#partial-load) |
+| `v32x4.load2(addr, offset) -> v128` | [v32x4.load2](portable-simd.md#partial-load) |
+| `v32x4.load3(addr, offset) -> v128` | [v32x4.load3](portable-simd.md#partial-load) |
```

Comment:
Partial loads and stores were [removed](https://github.com/kripken/emscripten/issues/4645#issuecomment-256101137) from asm.js since the non-power-of-2 size is awkward in some implementations.